Nordpool doc update (#36228)
home-assistant/home-assistant.github.io: source/_integrations/nordpool.markdown

| [peak/off-peak] time from       | Datetime          | The start date/time of the given timeframe.                                       |
| [peak/off-peak] time until      | Datetime          | The end date/time of the given timeframe.                                         |

The block price sensors are not enabled by default.

### Diagnostic sensors

| Sensor                    | Type              | Description                                                                       |
| ------------------------- | ----------------- | --------------------------------------------------------------------------------- |
| Currency                  | [Currency]        | The configured currency.                                                          |
| Exchange rate             | Integer           | The exchange rate between the configure currency and Euro's.                      |
| Last updated              | Datetime          | The time when the market prices were last updated.                                |

## Examples

A template sensor to add VAT and fixed cost is useful to get the actual energy cost in the energy dashboard. 

### UI Template

Create a helper using the UI.

> 1. Go to `Settings`, `Devices & Services` and choose the `Helpers` tab at the top.

Reviewer: ```suggestion
1. Go to {% my integrations title="**Settings** > **Devices & Services**" %} and at the top, choose the **Helpers** tab.
```

> 2. Select `Create helper` at the bottom right.

Reviewer: ```suggestion
2. In the bottom right corner, select **Create helper**.
```
we use bold to markup UI strings.

> 3. Choose `Template` and `Template a sensor`.

Reviewer: ```suggestion
3. Select **Template** and **Template a sensor**.
```

4. Enter the fields as shown below.

The template below takes the current price attributes, adds 0.1293 EUR as fixed costs and adds 21% VAT.

<p class='img'>
  <img src='/images/integrations/nordpool/nordpool_create_template.png' alt='Screenshot: Create template sensor'>
</p>

### YAML Template

A template sensor to add VAT and a fixed cost from an helper entity `input_number.add_fixed_cost`.

{% raw %}

```yaml
template:
  - sensor:
      - name: "Nordpool"
        unit_of_measurement: "EUR/kWh"
        state_class: measurement
        state: >
          # create a variable with the current price
          {% set cost = states('sensor.nord_pool_nl_current_price') | float(0) %}
          # create a variable with the additional fixed cost
          {% set add_cost = states('input_number.add_fixed_cost') | float(0) %}
          # Add cost and additional fixed cost. Add VAT (25%) by multiplying with 1.25 and round to 2 digits: 
          {{ ((cost + add_cost) * 1.25) | round(2, default=0) }}
```

{% endraw %}

### Energy Dashboard

> To use the Nordpool integration in the Energy dashboard, use the `Use an entity with current price` option when configuring grid consumption and production.

Reviewer: ```suggestion
To use the Nordpool integration in the **Energy** dashboard, when configuring grid consumption and production, use the **Use an entity with current price** option.
```

> <p class='img'>

Reviewer 2: I think we should use markdown to add images https://www.markdownguide.org/basic-syntax/#images

Reviewer: I think it's ok. I've seen html tags being used for images in other places.

Reviewer 2: In that case. If you're happy then approve and merge it 👍
